# DTU 10060 — E25 Physics Exam (2 June 2025)

Questions solved using `physics_tools`. Each question shows the full problem and options, followed by a computation cell and an explanation of the result.

In [1]:
import sys
import numpy as np
sys.path.insert(0, ".")
import physics_tools as pt

## Q1 — Statistical consistency of measurements

Earlier measurements of identical samples: 20.1 g, 20.2 g, 20.5 g, 19.8 g.  
A new sample measures **20.6 g**. Can we assume it has the same mass?

- A) Yes
- B) Maybe
- C) No

In [2]:
data    = [20.1, 20.2, 20.5, 19.8]
new_val = 20.6
result  = pt.is_consistent(new_val, data, n_sigma=2.0)
for k, v in result.items():
    print(f"{k}: {v}")

mean: 20.15
std: 0.2886751345948125
sem: 0.14433756729740624
n: 4
min: 19.8
max: 20.5
new: 20.6
deviation: 0.45000000000000284
sigma_dist: 1.5588457268120015
consistent: True


The sample mean is **20.15 g** with a standard deviation of **≈ 0.289 g**. The new measurement of 20.6 g deviates by ≈ 1.56σ from the mean — well within the 2σ consistency threshold. The value `consistent: True` confirms this.  

**Answer: A — Yes**

## Q2 — Power-law fit (log-log regression)

```python
k = np.array([1.2, 1.5, 2.2, 2.4, 3.4])
T = np.array([2.56, 2.29, 1.89, 1.81, 1.52])
```

$T \propto k^\alpha$. Find $\alpha$.

- A) α = −0.10
- B) α = −0.20
- C) α = −0.30
- D) α = −0.40
- E) α = −0.50
- F) α = −0.60
- G) α = −0.70
- H) α = −0.80
- I) α = −0.90
- J) α = −1.0

In [3]:
k   = np.array([1.2, 1.5, 2.2, 2.4, 3.4])
T   = np.array([2.56, 2.29, 1.89, 1.81, 1.52])
fit = pt.power_law_fit(k, T)
print(f"alpha = {fit['alpha']:.4f}")
print(f"R^2   = {fit['r2']:.4f}")

alpha = -0.5006
R^2   = 1.0000


The log-log regression returns α ≈ **−0.5006** and R² ≈ **1.000**, confirming an excellent power-law fit. Rounding to one decimal place gives α = −0.50, matching option E.

**Answer: E — α = −0.50**

## Q3 — Two-ball meeting height

Ball 1 is thrown vertically upward from the ground at $v_0 = 50\ \text{m/s}$.  
Ball 2 is dropped simultaneously from $h = 100\ \text{m}$ above Ball 1.  
At what height do they meet?

- A) 17.0 m
- B) 28.9 m
- C) 36.2 m
- D) 36.9 m
- E) 49.1 m
- F) 52.6 m
- G) 53.3 m
- H) 80.0 m
- I) 80.4 m
- J) 97.8 m

In [4]:
v0_ball1 = 50.0   # m/s upward
h_ball2  = 100.0  # m above ball 1

# y1(t) = v0*t - g*t^2/2,  y2(t) = 100 - g*t^2/2
# Setting equal: v0*t = 100  ->  t = 100/50 = 2 s  (gravity cancels)
t_meet = h_ball2 / v0_ball1
y_meet = pt.free_fall_y(y0=0.0, v0=v0_ball1, t=t_meet)
print(f"Meeting time  = {t_meet:.2f} s")
print(f"Meeting height = {y_meet:.2f} m")

Meeting time  = 2.00 s
Meeting height = 80.36 m


Because both balls experience the same gravitational acceleration, the $-\frac{1}{2}g t^2$ terms cancel when equating positions. The meeting condition reduces to $v_0 t = 100\ \text{m}$, giving $t = 2.00\ \text{s}$. Substituting back: $y = 50 \times 2 - \frac{1}{2}(9.82)(4) = 80.36\ \text{m} \approx 80.4\ \text{m}$.  

**Answer: I — 80.4 m**

## Q4 — River boat speed

Boat speed relative to water: $v$. Water current speed: $v_0$.  
Downstream travel of distance $L$: time $T$.  
Upstream return of distance $L$: time $2T$.  
Find $v$.

- A) $v = 4v_0$
- B) $v = 3v_0$
- C) $v = 2v_0$
- D) $v = v_0$
- E) $v = \frac{1}{2}v_0$
- F) $v = \frac{1}{3}v_0$
- G) $v = \frac{1}{4}v_0$

In [5]:
import sympy as sp

v, v0 = sp.symbols("v v0", positive=True)

# Downstream: L = (v + v0) * T
# Upstream:   L = (v - v0) * 2T
# Setting equal and dividing by T:
eq = sp.Eq(v + v0, 2 * (v - v0))
print("Equation:", eq)

sol = sp.solve(eq, v)[0]
print("v =", sol)

Equation: Eq(v + v0, 2*v - 2*v0)
v = 3*v0


Setting up $L = (v + v_0)T$ (downstream) and $L = (v - v_0) \cdot 2T$ (upstream), then dividing both sides by $T$ gives $(v + v_0) = 2(v - v_0)$. Expanding and collecting terms: $3v_0 = v$. The symbolic solver confirms $v = 3v_0$.

**Answer: B — $v = 3v_0$**

## Q5 — Bird dive: vertical acceleration graph

A bird flies horizontally at altitude, dives, then resumes horizontal flight near the ground.  
Which graph describes $a_y(t)$ (y-axis upward)?

*(Multiple graph options A–? shown in the original exam.)*

This is a qualitative question — no computation required.

During constant horizontal flight $a_y = 0$. To initiate the dive the bird accelerates downward ($a_y < 0$). To pull out and return to level flight the bird produces an upward acceleration ($a_y > 0$), then $a_y$ returns to 0. The correct graph shows the sequence: **0 → negative pulse → positive pulse → 0**.  

**Answer: C**

## Q6 — Centrifuge rotor radius

$\omega = 10\,000\ \text{rpm}$, centripetal acceleration $= 8500\,g$. Find the rotor radius $r$.

- A) 0.021 mm
- B) 7.8 mm
- C) 7.6 cm
- D) 15.2 cm
- E) 30 cm
- F) 2.6 m
- G) 74 m
- H) 0.27 km
- I) 0.55 km

In [6]:
r   = pt.centrifuge_radius(rpm=10000, a_in_g=8500)
omega = pt.rpm_to_rads(10000)
print(f"omega = {omega:.2f} rad/s")
print(f"r     = {r:.5f} m  =  {r * 100:.2f} cm")

omega = 1047.20 rad/s
r     = 0.07612 m  =  7.61 cm


$\omega = 10\,000 \times 2\pi/60 \approx 1047\\ \text{rad/s}$. From $a_c = \omega^2 r$ we get $r = 8500\,g / \omega^2 \approx 0.076\\ \text{m} = 7.6\\ \text{cm}$. This matches option C.  

**Answer: C — 7.6 cm**

## Q7 — Two stacked blocks: center-of-mass acceleration

Two blocks, each $m = 1.0\ \text{kg}$, stacked on a frictionless surface.  
$\mu_s = 0.80$, $\mu_k = 0.50$. Horizontal force $F = 20\ \text{N}$ applied to the **top** block.  
What is the acceleration of the common center of mass?

- A) 20 m/s²
- B) 15 m/s²
- C) 12 m/s²
- D) 10 m/s²
- E) 7.0 m/s²
- F) 5.1 m/s²
- G) 2.1 m/s²
- H) 0.18 m/s²

In [7]:
result = pt.two_blocks_stacked(
    m_top=1.0, m_bot=1.0, F=20.0, mu_s=0.80, mu_k=0.50, pulled="top"
)
print(f"slipping    = {result['slipping']}")
print(f"a_top       = {result['a_top']:.2f} m/s^2")
print(f"a_bot       = {result['a_bot']:.2f} m/s^2")
print(f"a_cm        = {result['a_cm']:.2f} m/s^2")
print(f"friction    = {result['friction']:.3f} N")
print(f"max static  = {0.80 * 1.0 * pt.g:.3f} N")

slipping    = True
a_top       = 15.09 m/s^2
a_bot       = 4.91 m/s^2
a_cm        = 10.00 m/s^2
friction    = 4.910 N
max static  = 7.856 N


The maximum static friction between the blocks is $\mu_s m g = 0.80 \times 1.0 \times 9.82 \approx 7.86\ \text{N}$. For both blocks to move together, the bottom block would need $F/2 = 10\ \text{N}$ — exceeding the static limit, so they slip. Regardless of slipping, the COM acceleration depends only on the net external force: $a_{cm} = F / (m_1 + m_2) = 20 / 2 = 10\ \text{m/s}^2$.  

**Answer: D — 10 m/s²**

## Q8 — Curling stone: kinetic friction coefficient

From the $v$–$t$ graph: the stone reaches approximately $2\ \text{m/s}$ and decelerates linearly to rest over approximately $20\ \text{s}$.  
Find $\mu_k$.

- A) $\mu_k \approx 0.01$
- B) $\mu_k \approx 0.02$
- C) $\mu_k \approx 0.03$
- D) $\mu_k \approx 0.04$
- E) $\mu_k \approx 0.05$
- F) $\mu_k \approx 0.06$
- G) $\mu_k \approx 0.07$
- H) $\mu_k \approx 0.08$
- I) $\mu_k \approx 0.09$

In [8]:
a_decel = 2.0 / 20.0  # |Delta v| / Delta t  from the graph
mu_k    = pt.mu_k_from_deceleration(a_decel)
print(f"deceleration = {a_decel:.3f} m/s^2")
print(f"mu_k         = {mu_k:.4f}")

deceleration = 0.100 m/s^2
mu_k         = 0.0102


Pure sliding friction gives $a = \mu_k g$, so $\mu_k = a/g = 0.10/9.82 \approx 0.0102$. This is closest to 0.01.  

**Answer: A — $\mu_k \approx 0.01$**

## Q9 — Pendulum string tension when horizontal

$R = 1.00\ \text{m}$, $m = 1.00\ \text{kg}$. The ball is given an initial horizontal speed $v = 10.0\ \text{m/s}$ with the string vertical (hanging down).  
What is the string tension when the string is **horizontal**?

- A) 0.0 N
- B) 9.82 N
- C) 21.7 N
- D) 32.1 N
- E) 44.0 N
- F) 80.4 N
- G) 98.2 N
- H) 132 N

In [9]:
result = pt.pendulum_tension(v0_bottom=10.0, R=1.00, theta_deg=90.0, m=1.00)
print(f"v at theta=90  = {result['v']:.4f} m/s")
print(f"height gained  = {result['h']:.4f} m")
print(f"Tension T      = {result['T']:.2f} N")

v at theta=90  = 8.9644 m/s
height gained  = 1.0000 m
Tension T      = 80.36 N


Energy conservation from the bottom to the horizontal position (height gain $= R = 1\ \text{m}$):  
$v_h^2 = v_0^2 - 2gR = 100 - 2(9.82)(1) = 80.36\ \text{m}^2/\text{s}^2$.  
At $\theta = 90°$ gravity acts perpendicular to the string, so the full centripetal acceleration is provided by tension alone:  
$T = m v_h^2 / R = 1.0 \times 80.36 / 1.0 \approx 80.4\ \text{N}$.  

**Answer: F — 80.4 N**

## Q10 — Spring-released blocks: momentum conservation

$m_A = 2.00\ \text{kg}$, $m_B = 6.00\ \text{kg}$. Both at rest; a compressed spring between them is released.  
$v_B = 1.00\ \text{m/s}$ (rightward). Find $v_A$.

- A) 0 m/s
- B) 1 m/s
- C) 2 m/s
- D) 3 m/s
- E) 4 m/s
- F) 5 m/s
- G) 6 m/s
- H) 9 m/s
- I) 12 m/s

In [10]:
vA = pt.explosion_2body(m1=2.00, m2=6.00, v2=1.00)
print(f"v_A = {vA:.2f} m/s  (negative = leftward)")
print(f"|v_A| = {abs(vA):.1f} m/s")

v_A = -3.00 m/s  (negative = leftward)
|v_A| = 3.0 m/s


Total momentum is zero (system released from rest), so $m_A v_A + m_B v_B = 0$.  
$v_A = -m_B v_B / m_A = -(6.00)(1.00)/2.00 = -3\ \text{m/s}$.  
The magnitude is **3 m/s** directed leftward.  

**Answer: D — 3 m/s**

## Q11 — Fusion energy distribution (He + neutron)

D + T $\rightarrow$ $^4$He $+$ n, $E_\text{fusion} = 17.6\ \text{MeV}$.  
$m_{\text{He}} = 4\, m_n$. Initial momenta $\approx 0$.  
How is the kinetic energy split?

- A) He 50%, n 50%
- B) He 80%, n 20%
- C) He 25%, n 75%
- D) He 20%, n 80%
- E) He 75%, n 25%
- F) He 65%, n 35%
- G) He 55%, n 45%

In [11]:
result = pt.fusion_energy_split(mass_ratio=4.0, E_total=17.6)
print(f"He (heavy): {result['heavy_fraction'] * 100:.0f}%   {result['E_heavy']:.2f} MeV")
print(f"n  (light): {result['light_fraction'] * 100:.0f}%   {result['E_light']:.2f} MeV")

He (heavy): 20%   3.52 MeV
n  (light): 80%   14.08 MeV


Momentum conservation from rest: $4m_n v_{\text{He}} = m_n v_n \Rightarrow v_n = 4 v_{\text{He}}$.  
KE ratio: $KE_\text{He}/KE_n = (4m_n v_\text{He}^2)/(m_n(4v_\text{He})^2) = 4/16 = 1/4$.  
So He carries $1/5 = 20\%$ (3.52 MeV) and the neutron carries $4/5 = 80\%$ (14.08 MeV).  

**Answer: D — He 20%, n 80%**

## Q12 — Bowling ball: sliding without rolling

A ball is released with horizontal velocity $v > 0$ and zero spin ($\omega = 0$). It moves rightward on a rough surface.  
Which statements are correct shortly after release?

- A) Acceleration is positive.
- B) Acceleration is negative.
- C) Angular acceleration is positive.
- D) Angular acceleration is negative.
- E) Friction is static.
- F) Friction is kinetic.
- G) $v = +r\omega$
- H) $v = -r\omega$
- I) $v \neq \pm r\omega$

In [12]:
result = pt.bowling_ball_after_release(m=1.0, r=0.11, v0=5.0, mu_k=0.3)
print(f"a     = {result['a']:.3f} m/s^2      (sign tells us B or A)")
print(f"alpha = {result['alpha']:.3f} rad/s^2  (sign tells us C or D)")
print(f"t_pure_rolling = {result['t_pure_rolling']:.3f} s  (> 0 means v != r*omega initially)")

a     = -2.946 m/s^2      (sign tells us B or A)
alpha = 66.955 rad/s^2  (sign tells us C or D)
t_pure_rolling = 0.485 s  (> 0 means v != r*omega initially)


- **a < 0 (B):** kinetic friction acts leftward, decelerating the ball.
- **α > 0 (C):** the friction torque spins the ball counterclockwise (positive sense), increasing ω.
- **Kinetic friction (F):** the ball is sliding ($v \neq r\omega$), so friction is kinetic, not static.
- **v ≠ ±rω (I):** at release ω = 0 while v > 0, so the rolling condition is not yet met; it is only satisfied after $t_{\text{roll}}$.

**Answer: B, C, F, I**

## Q13 — Work scaling with time

Frictionless floor. Constant force from rest, distance $d$ in time $t$ requires work $W_1$.  
Work $W_2$ to cover the **same** $d$ in time $3t$?

- A) $W_2 = W_1/81$
- B) $W_2 = W_1/9$
- C) $W_2 = W_1/3$
- D) $W_2 = W_1$
- E) $W_2 = 3W_1$
- F) $W_2 = 9W_1$
- G) $W_2 = 81W_1$

In [13]:
d, m_block = 10.0, 1.0  # arbitrary; only the ratio matters
r1 = pt.constant_F_to_travel_d_in_t(d=d, t=1.0, m=m_block)
r2 = pt.constant_F_to_travel_d_in_t(d=d, t=3.0, m=m_block)
print(f"W1      = {r1['W']:.4f} J")
print(f"W2      = {r2['W']:.4f} J")
print(f"W2/W1   = {r2['W'] / r1['W']:.6f}  (= 1/9 = {1/9:.6f})")

W1      = 200.0000 J
W2      = 22.2222 J
W2/W1   = 0.111111  (= 1/9 = 0.111111)


From $d = \frac{1}{2}at^2$ we get $a = 2d/t^2$, so $W = Fd = mad = 2md^2/t^2$.  
With three times the time: $W_2 = 2md^2/(3t)^2 = W_1/9$. The code confirms $W_2/W_1 = 1/9$ exactly.  

**Answer: B — $W_2 = W_1/9$**

## Q14 — Two discs: total mechanical energy vs time

A rotating disc has a stationary disc dropped onto it (no bounce). Friction between the discs brings them to a common angular velocity.  
Which graph describes $E_m(t)$?

*(Multiple graph options A–? shown in the original exam.)*

This is a qualitative question — no computation required.

- **Phase 1** (disc in free fall, no contact): total mechanical energy is conserved — $E_m$ is **flat**.
- **Phase 2** (impact): this is an inelastic collision — mechanical energy is lost instantaneously — $E_m$ **drops discontinuously**.
- **Phase 3** (kinetic friction between the discs as they equalise $\omega$): friction continuously dissipates rotational energy — $E_m$ **decreases gradually**.
- **Phase 4** (same $\omega$, no more slipping): no dissipation — $E_m$ is **flat** again.

The graph showing **flat → sharp drop → gradual decrease → flat** matches option H.  

**Answer: H**